docs/README.md

```diff
@@ -64,7 +64,7 @@ $  mvn clean install
 
 You could expect to see _BUILD SUCCESS_ at the end of the build console.
 
-![Alcor Build](modules/ROOT/images/alcor_build.JPG)
+![Alcor Build](modules/ROOT/images/alcor_build.png)
```

Comment:
This image looks a bit large. Can you adjust the size a bit? 

You can check the effect here:
https://github.com/yanmo96/alcor/blob/image_update/docs/README.md

Reply:
Just updated the readme file to limit to 720px. Hope it looks ok now :)